scripts/mutect2_wdl/unsupported/m2_basic_validation.wdl

```diff
@@ -0,0 +1,363 @@
+import "mutect2.wdl" as m2
+
+workflow m2_validation {
+
+    #### M2 parameters
+    String gatk4_jar
+    File? intervals
+    File ref_fasta
+    File ref_fasta_index
+    File ref_dict
+    File tumor_bam
+    File tumor_bam_index
+    String tumor_sample_name
+    File? normal_bam
+    File? normal_bam_index
+    String? normal_sample_name
+    File? pon
+    File? pon_index
+    Int scatter_count
+    File? gnomad
+    File? gnomad_index
+    File? variants_for_contamination
+    File? variants_for_contamination_index
+    Boolean is_run_orientation_bias_filter
+    Boolean is_run_oncotator
+    String m2_docker
+    String basic_bash_docker = "ubuntu:16.04"
+    String oncotator_docker
+    File? gatk4_jar_override
+    Int preemptible_attempts
+    File? onco_ds_tar_gz
+    String? onco_ds_local_db_dir
+    Array[String] artifact_modes
+    File picard_jar
+    String? m2_extra_args
+    String? m2_extra_filtering_args
+    String? sequencing_center
+    String? sequence_source
+    File? default_config_file
+    Boolean is_bamOut = false
+    #####
+
+    ### parameter-fu
+    ### Discovery bams
+    File tumor_bam_file_list
+    Array[File] tumor_bam_files = read_lines(tumor_bam_file_list)
+
+    File tumor_bam_file_index_list
+    Array[File] tumor_bam_indices = read_lines(tumor_bam_file_index_list)
+
+    File normal_bam_file_list
+    Array[File] normal_bam_files = read_lines(normal_bam_file_list)
+
+    File normal_bam_file_index_list
+    Array[File] normal_bam_indices = read_lines(normal_bam_file_index_list)
+
+    ### validation bams
+    File validation_tumor_bam_file_list
+    Array[File] validation_tumor_bam_files = read_lines(validation_tumor_bam_file_list)
+
+    File validation_tumor_bam_file_index_list
+    Array[File] validation_tumor_bam_indices = read_lines(validation_tumor_bam_file_index_list)
+
+    File validation_normal_bam_file_list
+    Array[File] validation_normal_bam_files = read_lines(validation_normal_bam_file_list)
+
+    File validation_normal_bam_file_index_list
+    Array[File] validation_normal_bam_indices = read_lines(validation_normal_bam_file_index_list)
+    #####
+
+    # A name to identify this group of samples.  This can be arbitrary, but should not contain special characters, nor whitespace.
+    String group_id
+
+
+    scatter (i in range(length(tumor_bam_files))) {
```

Comment:
Because that has a custom tsv that I'd have to generate.  As an aside, I'd like to move the multi sample WDL to use the same convention that this uses.

No action.

Reply:
> I'd like to move the multi sample WDL to use the same convention that this uses.

Agreed -- it's issue #3387.